rest/server_context_test.go

```diff
@@ -632,8 +633,20 @@ func TestLogFlush(t *testing.T) {
 			// Flush memory loggers
 			base.FlushLoggerBuffers()
 
+			// Concurrent calls to FlushLogBuffers should not cause data race or wait group reuse issues
+			// Wait for concurrent calls so they don't cause issues with SetupAndValidateLogging of next t.Run
+			var flushCallsWg = sync.WaitGroup{}
+			for i := 0; i < 5; i++ {
+				flushCallsWg.Add(1)
+				go func() {
+					base.FlushLogBuffers()
+					flushCallsWg.Done()
+				}()
+			}
+
 			// Flush collation buffers to ensure the files that will be built do get written
 			base.FlushLogBuffers()
+			flushCallsWg.Wait()
```

Comment:
Is the additional flush (on line 648) still needed now that we're flushing as part of the concurrency checking?

Reply:
True... also bumped number of concurrent calls to make sure we hit wait re-use issue more easily, after more testing locally.